modules/ROOT/pages/download-and-install-service-mesh.adoc

```diff
@@ -0,0 +1,80 @@
+= Download and Install Service Mesh
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: service mesh, microservices, downloading, installing
+
+image::install-service-mesh-breadcrumb.png[]
+
+Before you install Service Mesh, you must first download the product from the installer download URL. As part of installation, the following required components are deployed on a Kubernetes cluster:
```

Comment:
I don't think we need to say "Before you install Service Mesh, you must first download ". It's obvious that you download before you install. 
I think this whole line can be just removed. "Before you install Service Mesh, you must first download the product from the installer download URL

Reply:
@priyaayyar this section is still not fixed in the latest PR